### DISCUSSION TITLE
tinker plugin via phpstorm JetBrains Client

### DISCUSSION THREAD

**Discussion by s4muel** [Roboroads/laravel-tinker#282] [Everything Else]
hi there, @Roboroads, thanks for your plugin, works well in the phpstorm.

but when i use phpstorm remotely via JetBrains Client (2023.2.2 RC), i cannot find your plugin in the marketplace.
i understand it is a different "product". on the plugin page (https://plugins.jetbrains.com/plugin/14957-laravel-tinker) it says:
```
Not compatible with the version of your running IDE (Gateway 2023.2.2, Client 2023.2.2 RC)
Compatible with IntelliJ IDEA Ultimate, PhpStorm
```
is there any possibility to run it in this remote client?

**Comment by Roboroads** [comment]
Does the JB Client have the php plugin installed? If yes, it might work! You can [download the latest release](https://github.com/Roboroads/laravel-tinker/releases/tag/v2.6.1) and install it manually. Let me know if that works :)

**Reply by s4muel** [reply]
thanks for the quick reply and the tip, i tried it, looked promising, but then i found a dead end i think:
![tinker](https://github.com/Roboroads/laravel-tinker/assets/430949/b10fe275-0825-4fec-9af0-f58fbeb98cd4)

**Reply by Roboroads** [reply]
Yea that's a no - sorry :(
